app/lib/features/member/dialogs/show_change_power_level_dialog.dart

```diff
@@ -114,13 +111,11 @@ class __ChangePowerLevelDialogState extends State<_ChangePowerLevelDialog> {
                   // Only numbers
                   validator: (String? value) {
                     if (currentMemberStatus == 'Custom') {
-                      if (value == null || int.tryParse(value) == null) {
-                        return L10n.of(context)
-                            .youNeedToEnterCustomValueAsNumber;
+                      if ((value == null || int.tryParse(value) == null)) {
+                        return L10n.of(context).youNeedToEnterCustomValueAsNumber;
```

Comment:
I think a recent update reformatted this code and now this reverts that ...
```suggestion
                      if (value == null || int.tryParse(value) == null) {
                        return L10n.of(context).youNeedToEnterCustomValueAsNumber;
```